frontendservice/src/router/index.js

```diff
@@ -15,8 +15,6 @@ import firebase from "firebase";
 import Vue from "vue";
 import VueRouter from "vue-router";
 
-//ToDo: kschauhan - import vue components, edit paths and router when more .vue files are pushed
```

Comment:
Nit. TODO instead of ToDo.

Reply:
Will remember this for future. In this particular commit, I have removed this particular TODO as its work is over.